x-pack/plugins/cases/server/saved_object_types/migrations/cases.test.ts

```diff
@@ -577,4 +580,46 @@ describe('case migrations', () => {
       });
     });
   });
+
+  describe('update severity', () => {
+    for (const oldSeverityValue of Object.values(CaseSeverity)) {
+      it(`migrates ${oldSeverityValue} severity string label to matching number`, () => {
+        const doc = {
+          id: '123',
+          type: 'abc',
+          attributes: {
+            severity: oldSeverityValue,
+          },
+          references: [],
+        } as unknown as SavedObjectUnsanitizedDoc<CaseAttributes>;
+
+        expect(convertSeverity(doc)).toEqual({
+          ...doc,
+          attributes: {
+            ...doc.attributes,
+            severity: SEVERITY_EXTERNAL_TO_ESMODEL[oldSeverityValue],
```

Comment:
nit: I think we should hardcode the expected value and not rely on the `SEVERITY_EXTERNAL_TO_ESMODEL` (same for other tests below). If there is a bug in `SEVERITY_EXTERNAL_TO_ESMODEL` the test will always pass. You can do `for (const [oldSeverityValue, expectedValue] of [['low', 10], ...])`.